app/lib/actions/pulp3/repository/update_distributions.rb

```diff
@@ -0,0 +1,16 @@
+module Actions
+  module Pulp3
+    module Repository
+      class UpdateDistributions < Pulp3::Abstract
+        def plan(repository, smart_proxy)
+          plan_self(:repository_id => repository.id, :smart_proxy_id => smart_proxy.id) if repository.root.url?
+        end
+
+        def run
+          repo = ::Katello::Repository.find(input[:repository_id])
+          output[:response] = repo.backend_service(smart_proxy).update_distributions
```

Comment:
can we just call refresh_distributions and not have an aliased method (essentially)